File: Solutions/2206. Divide Array Into Equal Pairs/2206. Divide Array Into Equal Pairs.go
func divideArray(nums []int) bool {
    m:=make(map[int]int)
    for _,n:=range nums{
        m[n]+=1
    }
    for _,v:=range m{
        if v%2!=0{
            return false
        }
    }
    return true
}

File: Solutions/2206. Divide Array Into Equal Pairs/2206. Divide Array Into Equal Pairs.cpp
class Solution {
public:
    bool divideArray(vector<int>& nums) {
        unordered_map<int,int> um;
        for(auto& n:nums){
            ++um[n];
        }
        for(auto& [k,v]:um){
            if(v%2!=0){
                return false;
            }
        }
        return true;
    }
};